### DISCUSSION TITLE
Encoding single quote

### DISCUSSION THREAD

**Discussion by leodip** [microcosm-cc/bluemonday#199] [Q&A]
I noticed this behaviour:

```go
p := bluemonday.StrictPolicy()
p.AllowStandardURLs()

sanitized := p.Sanitize("O'Reilly")
slog.Info(sanitized) // ouputs "O&#39;Reilly"
```

It's converting the single quote (or apostrophe) into an HTML entity.

Is there a way to prevent it from doing that? I want to keep the apostrophe.

**Comment by gsgghotra** [comment]
There's _**_EscapeString function_**_ in html package. First, you will need to` import "html"` and then try to use **html.UnescapeString** 


`slog.Info(html.UnescapeString(sanitized))`

Its either `html.UnescapeString` or `html.EscapeString`
